### PR TITLE
docs(cosmic-swingset): Clean up the readme

### PR DESCRIPTION
**Pull request by gibson042**
## Description

Remove/replace outdated content. I think what remains is _still_ incorrect (e.g., no scenario seems to include a Pixel Demo), but should at least be improved from current state.

### Security Considerations

n/a

### Documentation Considerations

n/a

### Testing Considerations

n/a

### CODE REVIEW

**Comment by JimLarson** [comment]
I'll have to defer to mfig for reviewing this.

File: packages/cosmic-swingset/README.md [hunk 1]
```diff
@@ -148,25 +146,33 @@ $ make scenario0-setup
...
 
+And to interact with them:
 ```js
-home.uploads~.list()
-home.uploads~.get('encouragementBot')~.spawn()~.encourageMe('Person')
+E(
+  E(
+    E(home.uploads).get('encouragementBot')
+  ).spawn()
+).encourageMe('Person')
```
Comment (michaelfig) [inline packages/cosmic-swingset/README.md]: This section should be deleted.

File: packages/cosmic-swingset/README.md [hunk 2]
```diff
@@ -148,25 +146,33 @@ $ make scenario0-setup
...
 ### Initial Endowments
 
-When a client is started up, it has a few items in a record named home.
+When a client is started up, it has a few items in a record named `home`.
 * sharingService: a service that makes it possible to pass capabilities between vats
 * canvasStatePublisher: a service with the message subscribe(callback)
-* [uploads](./lib/ag-solo/contracts/README-contract.md): a private directory
+* uploads: a private directory
  of contracts you've uploaded
 * registry: a public directory for published objects
```
Comment (michaelfig) [inline packages/cosmic-swingset/README.md]: These are all deleted.

**Review by michaelfig — COMMENTED** [review]
Just some recommended changes.  Ping me again to rereview.

File: packages/cosmic-swingset/README.md (hunk 1 above)
Comment (gibson042) [reply packages/cosmic-swingset/README.md]: Just this code block, or the one above it as well, or even more than that?

File: packages/cosmic-swingset/README.md (hunk 2 above)
Comment (gibson042) [reply packages/cosmic-swingset/README.md]: Just this line, or the entirety of Initial Endowments?

Comment (michaelfig) [reply packages/cosmic-swingset/README.md]: The part that I highlighted: `* canvasStatePublisher` up to and including `* registry`.

File: packages/cosmic-swingset/README.md (hunk 1 above)
Comment (michaelfig) [reply packages/cosmic-swingset/README.md]: The part that I highlighted: `To see the contracts ...` up to and including `).encourageMe('Person')`.

**Comment by gibson042** [comment]
Thanks @michaelfig, back to you.

**Review by michaelfig — APPROVED** [review]

**Pull request merged**